pkg/network/encoding/marshal/modeler_test.go

```diff
@@ -1,67 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-
-package marshal
-
-import (
-	"strconv"
-	"sync"
-	"testing"
-
-	model "github.com/DataDog/agent-payload/v5/process"
-	"github.com/stretchr/testify/assert"
-
-	"github.com/DataDog/datadog-agent/pkg/config/mock"
-	"github.com/DataDog/datadog-agent/pkg/network"
-)
-
-func TestConnectionModelerAgentConfiguration(t *testing.T) {
-	tests := []struct {
-		npm, usm, ccm, csm bool
```

Comment:
Can we keep this test with `csm` removed?

Reply:
:+1: